firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -207,6 +208,7 @@ public synchronized void onActivityResumed(Activity activity) {
       updateAppState(ApplicationProcessState.FOREGROUND);
       if (mIsColdStart) {
         // case 1: app startup.
+        sendAppColdStartUpdate();
```

Comment:
Although I have concerns on how we measure our cold startup time and I don't think initializing FirebasePerformance here will actually solve the startup time issue. However, I do agree with the current issue with dropbox and the traces provided by them (_which does not include `onResume()` as part of the measurements which is strange to me_) and this might help solve their use-case but not for all.

As discussed for now it's ok to rely on cold startup callback here but this need to change.

Reply:
Agree. The main reason that I include initialization code here is because our Fireperf's definition of [app startup time](https://firebase.google.com/docs/perf-mon/app-start-foreground-background-traces?platform=android#app-start), which does not count onResume() for app startup time. We might need to update how we measure app startup time based on Android vitals's definition.

Reply:
Filed b/183657588 for this.